test/global/activate/git_package_test.dart

```diff
@@ -9,9 +9,9 @@ import '../../descriptor.dart' as d;
 import '../../test_pub.dart';
 
 void main() {
-  test('activates a package from a Git repo', () async {
-    ensureGit();
+  setUp(ensureGit);
```

Comment:
Is this something we can do in a separate PR that's super easy to land -- just to reduce noise in this PR?

Reply:
The noise is removed now :)